packages/peer-store/test/save.spec.ts

```diff
@@ -67,12 +67,13 @@ describe('save', () => {
     expect(peer.addresses).to.deep.equal(
       supportedMultiaddrs.map((multiaddr) => ({
         isCertified: false,
-        multiaddr
+        multiaddr: multiaddr.encapsulate(`/p2p/${otherPeerId.toString()}`)
       }))
     )
     expect(previous).to.be.undefined()
   })
 
+  // todo: this test needs fixed: it's not testing what it says it is; or it's passing when it shouldn't
   it('emits on set if not storing the exact same content', async () => {
```

Comment:
I left this comment in the code to trigger a discussion. This test says it's emitting when not storing the same content, but I see we have `patch` and other methods now. There are two potential actions here:

1. rename the test if not fully covered by others (I believe it is)
2. figure out why the `// set 2` is not throwing (was patch added, and this is no longer the expectation?)
3. remove the test entirely

Reply:
Moved this to https://github.com/libp2p/js-libp2p/issues/1962